OneSchedule.Domain.Abstractions/OneSchedule.Domain.Abstractions.csproj

```diff
@@ -1,11 +1,12 @@
-<Project Sdk="Microsoft.NET.Sdk">
+﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
     <TargetFramework>net5.0</TargetFramework>
   </PropertyGroup>
 
   <ItemGroup>
     <PackageReference Include="MongoDB.Driver" Version="2.13.1" />
+    <PackageReference Include="Telegram.Bot" Version="16.0.2" />
```

Comment:
This package was added to all projects. It should be added only to that project which uses this package